netutils/ip.py

```diff
@@ -247,3 +272,25 @@ def get_usable_range(ip_network):
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+def count_bits(i: int) -> int:
```

Comment:
Should this be a private method? 

Also, should avoid using single letter variable names.

Reply:
Up to you.